Algorithms/Sequences/A000040_PrimesSequence.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+
+namespace Algorithms.Sequences
+{
+    /// <summary>
+    /// <para>
+    ///     Sequence of prime numbers.
+    /// </para>
+    /// <para>
+    ///     Wikipedia: https://wikipedia.org/wiki/Prime_number.
+    /// </para>
+    /// <para>
+    ///     OEIS: https://oeis.org/A000040.
+    /// </para>
+    /// </summary>
+    public class A000040_PrimesSequence : ISequence
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename class 'A000040_PrimesSequence' to match pascal case naming rules, consider using 'A000040PrimesSequence'.](https://app.codacy.com/app/siriak/C-Sharp/pullRequest?prid=4650688)